src/app/story/story.component.html

```diff
@@ -6,7 +6,7 @@
     <ng-container *ngIf="story">
       <app-storyrow [node]="node" [backwardSiblings]="story.meta.story_related_nodes_backward[i]"
           [forwardSiblings]="story.meta.story_related_nodes_forward[i]" [last]="last"
-          [active]="node === selectedNode"
+          [active]="node === selectedNode" [waitTime]="waitTimes[i]"
```

Comment:
Could you please use `[waitTime]="story.meta.wait_times[i]"` instead? That way you can avoid the unneeded class property of `waitTimes`.